canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.scss

```diff
@@ -138,7 +138,16 @@
 			// ReadonlyTable with single row selection is non-interactive. Nothing should happen on hover
 			background-color: $layer-hover-01;
 		}
-
+		.properties-vt-row-doubleClick{
```

Comment:
We should have row focus when a user tabs into the row but I don't think we need something special for double click.  Also, active and focus are different things.